app/components/App.js

```diff
@@ -15,6 +16,26 @@ class App extends Component {
     this._getSaveData = this._getSaveData.bind(this);
   }
 
+  componentDidMount() {
+    this._captureClicks(this.props.state.cmsStatus);
+  }
+
+  componentDidUpdate() {
+    this._captureClicks(this.props.state.cmsStatus);
+  }
+
+  _captureClicks(cmsStatus) {
+    if ('cms.isSaving' === cmsStatus) {
+      const cover = document.getElementById('appCover');
+      if (cover) {
+        cover.addEventListener('click', (evt) => {
+          evt.preventDefault();
+          evt.stopPropagation();
```

Comment:
Took me a moment to realize what you are doing here, please add a comment as to what/why.